### PR TITLE
Add custom class, remove category line features

### PR DESCRIPTION
**Pull request by kispali**

### CODE REVIEW

**Review by eddichen — APPROVED** [review]
Approved FE parts

**Review by eddichen — APPROVED** [review]

File: src/MX/MegaMenu/Block/Adminhtml/Menu/Edit/Form.php [hunk 1]
```diff
@@ -239,6 +249,17 @@ protected function _prepareForm()
                 'name' => 'content_category_type',
             ]
         );
+        $content->addField(
+            'remove_category_anchor',
+            'MX\MegaMenu\Data\Form\Element\Toggle',
```
Comment (tkotosz) [inline src/MX/MegaMenu/Block/Adminhtml/Menu/Edit/Form.php]: :point_up: import class pls

File: src/MX/MegaMenu/Model/Menu.php [hunk 2]
```diff
@@ -345,13 +345,43 @@ public function getProcessedMenuItems()
             $parentId = $item['menu_item_parent_id'];
 
             $menuItem = $this->menuItemFactory->create();
+            $menuItemData = $menuItem->getItemData($item);
+
             if ($item['menu_item_parent_id'] == 0) {
```
Comment (tkotosz) [inline src/MX/MegaMenu/Model/Menu.php]: $parentId == 0

**Review by tkotosz — CHANGES_REQUESTED** [review]

File: src/MX/MegaMenu/Block/Adminhtml/Menu/Edit/Form.php (hunk 1 above)
Comment (kispali) [reply src/MX/MegaMenu/Block/Adminhtml/Menu/Edit/Form.php]: sorted

File: src/MX/MegaMenu/Model/Menu.php (hunk 2 above)
Comment (kispali) [reply src/MX/MegaMenu/Model/Menu.php]: sorted

**Review by tkotosz — APPROVED** [review]

**Pull request merged**